### PR TITLE
IA reader nav: tabs order and naming

### PR DESCRIPTION
**Pull request by develric**
This PR is one of a set of PRs which aim to supersede the #10716 decomposing it in more manageable/testable/reviewable PRs (cc @osullivanchris ).

The current one is based and requires the #10782 

We have identified in the Reader 4 main tabs that we want to be in the following order and naming:

1. FOLLOWING
2. DISCOVER
3. LIKES
4. SAVED

This PR name the relevant tabs with the above naming and put them in front of eventual other tabs. Using string resources to account for i18n.

In the following an example

<p align="center">
  
</p>
 
## To test

1. Open the reader and check that you get a list of item with correct names and order.
2. Select a tab, close/open the app and check you get the same correct tab selected.
3. Smoke test the reader as done in # 10782 test steps.

PR submission checklist:

- [x] I have considered adding unit tests where possible.

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java [hunk 1]
```diff
@@ -2338,6 +2340,100 @@ public static void resetLastUpdateDate() {
     private class LoadTagsTask extends AsyncTask<Void, Void, ReaderTagList> {
         private final FilteredRecyclerView.FilterCriteriaAsyncLoaderListener mFilterCriteriaLoaderListener;
 
+        private static final String KEY_FOLLOWING = "following";
+        private static final String KEY_DISCOVER = "discover";
+        private static final String KEY_LIKES = "likes";
+        private static final String KEY_SAVED = "saved";
+
+        private class TagInfo {
```
Comment (planarvoid) [inline WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java]: Would it maybe be possible to extract this logic to a separate class and just expose `getOrderedTagsList`?

**Comment by develric** [comment]
Hi @planarvoid, thanks for the review 😊 , I created the TagInfo dedicated class and was looking into moving some of the logic in a separate class also when I found that basically the class I was trying to create was the ReaderUtils...it is a static based utility class so not the best for testability, but I moved the logic there. Maybe we can consider to create a dedicated task to port all the ReaderUtils static methods to a not static kotlin utility class later. Wdyt?

**Comment by planarvoid** [comment]
hey @develric 👋 
> I found that basically the class I was trying to create was the ReaderUtils
I think this is fine since it's only called from a place which is not tested, it doesn't need to be injected. You can still write tests for the static methods in the ReaderUtils though :-)

**Comment by develric** [comment]
Hi @planarvoid makes sense...added really basic unit test here a827b591d38e74e2c885c35e177305c9d9420634 😊

**Pull request merged**